client-core/src/wallet/default_wallet_client.rs

```diff
@@ -1134,6 +1136,28 @@ mod tests {
             PublicKey::from(&PrivateKey::new().unwrap()),
         ];
 
```

Comment:
assert!(public_keys.size()>0);

Reply:
The `public_keys` size check is performed in the underlying function
https://github.com/calvinaco/chain/blob/feature/cro-313-create-multisig-address-rpc/client-core/src/service/root_hash_service.rs#L49